src/main/java/teammates/ui/output/FeedbackSessionLogData.java

```diff
@@ -5,13 +5,13 @@
 /**
  * The response log of a single feedback session.
  */
-public class FeedbackSessionLog {
+public class FeedbackSessionLogData {
```

Comment:
Going by the original API design would require the use of these three files -- these had been defined previously in https://github.com/TEAMMATES/teammates/pull/10978
One possibility is to drop `FeedbackSessionLogsData`, if we change it such that the frontend is limited to retrieving logs for one feedback session at a time only. This would reduce the number of files, but it would correspondingly require a change in the API